content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,567 @@
+your own OpenTelemetry collector binary. You can add or remove components
+(receivers, processors, connectors and exporters) to suit your needs.
+
+1.  Follow the OpenTelemetry Collector Builder
+    [installation instructions](/docs/collector/custom-collector/).
+
+2.  Write a Configuration File:
+
+    Once installed, the next step is to create a configuration file
+    `builder-config.yaml`. This file defines the collector components you want
+    to include in your custom binary.
+
+    Here is an example of the configuration file you can use featuring your new
+    connector component:
+
+        dist:
+            name: otelcol-dev-bin
+            description: Basic OpenTelemetry collector distribution for Developers
+            output_path: ./otelcol-dev
+            otelcol_version: 0.81.0
+
+
+        exporters:
+            - gomod:
+            go.opentelemetry.io/collector/exporter/loggingexporter v0.81.0
+
+
+        processors:
+            - gomod:
+            go.opentelemetry.io/collector/processor/batchprocessor v0.81.0
+
+
+        receivers:
+            - gomod:
+        go.opentelemetry.io/collector/receiver/otlpreceiver v0.81.0
+
+
+        connectors:
+            - gomod: github.com/gord02/exampleconnector v0.81.0
+
+
+        replaces:
+        # a list of "replaces" directives that will be part of the resulting go.mod
+
+        # This replace statement is necessary since the newly added component is not found/published to GitHub yet. Replace references to GitHub path with the local path
+        - github.com/gord02/exampleconnector => [PATH-TO-COMPONENT-CODE]/exampleconnector
+
+    It is necessary to include a replace statement. The replace section since
+    your newly created component is not published to GitHub yet. The references
+    to the GitHub path for your component will need to be replaced with the
+    local path to your code.
+
+    There are further details on replacement in go at
+    [Go mod file Replace](https://go.dev/ref/mod#go-mod-file-replace).
+
+3.  Build Your collector binary:
+
+    Run the builder while passing in the builder config file detailing the
+    included connector component which will then build the custom collector
+    binary:
+
+        builder --config [PATH-TO-CONFIG]/builder-config.yaml
+
+    This will generate the collector binary in the specified output path
+    directory that was in your config file.
+
+4.  Run Your collector binary:
+
+    Now you can run your custom collector binary:
+
+    ```sh
+    ./[OUTPUT_PATH]/[NAME-OF-DIST] --config [PATH-TO-CONFIG]/config.yaml
+    ```
+
+    The output path name and name of dist is detailed in the
+    `build-config.yaml`.
+
+Additional resources on the OpenTelemetry Collector Builder:
+
+- [Building a custom collector](/docs/collector/custom-collector)
+- [OpenTelemetry Collector Builder README](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder)
```

Comment:
You might also consider linking to the connectors readme: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md